src/Calculator/Controls/OverflowTextBlock.cpp

```diff
@@ -64,6 +64,12 @@ void OverflowTextBlock::OnApplyTemplate()
         m_itemsControl = safe_cast<ItemsControl ^>(uiElement);
     }
 
+    uiElement = GetTemplateChild("EditableToken");
```

Comment:
Wrong PR?

Reply:
This is part of the styling for Always-on-Top.